### PR TITLE
Features menu small improvements

### PR DESCRIPTION
**Pull request by Saveli0S**

### CODE REVIEW

File: src/components/ProcessIntegration/ProcessIntegration.jsx [hunk 1]
```diff
@@ -30,24 +30,26 @@ export const icons = [
...
-            {slide.icon}
-          </div>
-        ))}
-      </Marquee>
-    </div>
-  </section>
-);
+export const ProcessIntegration = forwardRef(function ProcessIntegration(_, ref) {
+  return (
+    <section className={cx(getBlocksWith())} ref={ref}>
```
Comment (ViktorSoroka07) [inline src/components/ProcessIntegration/ProcessIntegration.jsx]:
```suggestion
export const ProcessIntegration = forwardRef((props, ref) => <section className={cx(getBlocksWith())} ref={ref}>
```

File: src/components/Features/Features.jsx [hunk 2]
```diff
@@ -65,20 +65,47 @@ export const Features = () => {
...
+    const observer = new IntersectionObserver((entries) => {
+      entries.forEach((entry) => {
+        const scrollTop = document.documentElement.scrollTop;
+        const isPositionOnIntegrationElem = processIntegrationRef.current.offsetTop < scrollTop;
+
+        if (entry.isIntersecting || isPositionOnIntegrationElem) {
+          setIsFeaturesMenuSticky(false);
+        } else {
+          setIsFeaturesMenuSticky(true);
+        }
```
Comment (ViktorSoroka07) [inline src/components/Features/Features.jsx]:
```suggestion
        setIsFeaturesMenuSticky(!(entry.isIntersecting || isPositionOnIntegrationElem));
```

File: src/components/Features/Features.jsx [hunk 3]
```diff
@@ -65,20 +65,47 @@ export const Features = () => {
...
+  const headerHeight = 76;
   const stickyScrollTopPosition = 1200;
   const featuresBlockStickyPositionWithHeader = featuresBlockStickyPosition - headerHeight;
   const menuItemActiveClassName = getBlocksWith('__features-navigation-item--active');
   const featureItemClassName = getBlocksWith('__features-navigation-item');
 
   const setHistoryValue = (val) => window.history.replaceState(null, '', `/features/${val}`);
 
+  useEffect(() => {
+    const observer = new IntersectionObserver((entries) => {
```
Comment (ViktorSoroka07) [inline src/components/Features/Features.jsx]: First time I see this used in codebase, good to have this use case😀

File: src/components/ProcessIntegration/ProcessIntegration.jsx (hunk 1 above)
Comment (Saveli0S) [reply src/components/ProcessIntegration/ProcessIntegration.jsx]: Have the linter error if still use arrowFn
image

File: src/components/Features/Features.jsx (hunk 2 above)
Comment (Saveli0S) [reply src/components/Features/Features.jsx]: replaced

File: src/components/ProcessIntegration/ProcessIntegration.jsx (hunk 1 above)
Comment (ViktorSoroka07) [reply src/components/ProcessIntegration/ProcessIntegration.jsx]: In this case we can do the following

ProcessIntegration.displayName = 'ProcessIntegration';

Comment (Saveli0S) [reply src/components/ProcessIntegration/ProcessIntegration.jsx]: done

**Review by ViktorSoroka07 — DISMISSED** [review]

**Review by ViktorSoroka07 — APPROVED** [review]

**Pull request merged**